backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/floatingnotification/internal/BpkFloatingNotification.kt

```diff
@@ -110,21 +111,16 @@ internal fun floatingNotificationTransforms(): AnimatedContentScope<BpkFloatingN
   }
 
 private object BpkFloatingNotificationSizes {
-  const val SMALL_MOBILE_MAX_WIDTH = 360
-  const val MOBILE_MAX_WIDTH = 512
-
-  val SmallMobile = DpSize(288.dp, 52.dp)
-  val Mobile = DpSize(312.dp, 52.dp)
-  val Tablet = DpSize(400.dp, 72.dp)
+  const val TABLET_MIN_WIDTH = 769
+  val PhoneAndSmallTableWidth = 52.dp
+  val TabletAndLarger = DpSize(400.dp, 72.dp)
 }
 
 private fun Modifier.floatingNotificationSize(configuration: Configuration): Modifier =
   when {
-    configuration.screenWidthDp < BpkFloatingNotificationSizes.SMALL_MOBILE_MAX_WIDTH ->
```

Comment:
is it possible to use android native apis instead of comparing pixel sizes?
I'm a bit rusty on Android APIs haha but I generally think that checking pixels is almost ever a scalable solution

Reply:
This is how designers have implemented in Figma to keep consistency between platforms.

Reply:
You're right, it's not a scalable solution. However, this is what Android has been using as a platform solution